app/views.py

```diff
@@ -210,6 +219,10 @@ def registration(request):
                 del form.cleaned_data['location_lat']
                 del form.cleaned_data['location_lng']
                 del form.cleaned_data['location_name']
+                
+                #Remove information no store 
```

Comment:
Creo que debería decir "Remove information to store"

Reply:
done

Reply:
No veo el cambio

Reply:
Esta oudated .Si entras al codigoo se ven los cambios

![Screenshot from 2021-06-14 17-34-18](https://user-images.githubusercontent.com/12649261/121962555-e4791280-cd36-11eb-81cb-6456c352930d.png)